tee-worker/ts-tests/integration-tests/identity.test.ts

```diff
@@ -135,6 +135,7 @@ describeLitentry('Test Identity', 0, (context) => {
 
         // Bob links:
         // - alice's substrate identity
+        // @FIXME: comment says alice; code says charlie. Which one is it supposed to be !?
         const aliceSubstrateIdentity = await buildIdentityHelper(
             u8aToHex(context.substrateWallet.charlie.addressRaw),
```

Comment:
@Verin1005 I left this question there; do you know the answer?

Reply:
Naming error, it's clearly supposed to be 'charlie.' I'll make the correction here.

Reply:
fixed!